testng-core/src/main/java/org/testng/TestNG.java

```diff
@@ -317,32 +317,13 @@ private OverrideProcessor getProcessor() {
     return new OverrideProcessor(m_includedGroups, m_excludedGroups);
   }
 
-  private void parseSuite(String suitePath) {
+  private Collection<XmlSuite> parseSuite(String suitePath) {
```

Comment:
Is it possible to make it static?

Reply:
No, it uses `getProcessor` method, which operates members.